framework/src/consumption/examples/opensearch-api.lit.ts

```diff
@@ -8,23 +8,22 @@ class ExampleOpenSearchApiStack extends cdk.Stack {
   constructor(scope: Construct, id: string , props:cdk.StackProps) {
 
     super(scope, id, props);
+    this.node.setContext('@data-solutions-framework-on-aws/removeDataOnDestroy', true);
     /// !show
     const osCluster = new dsf.consumption.OpenSearchCluster(this, 'MyOpenSearchCluster',{
```

Comment:
Can you change this to how the construct should be initialized, and not testing it through the openseach cluster construct. We have an example on kafka-api unit tests `kafka-api.test.ts`